cmd/juju/application/store/charmadapter_test.go

```diff
@@ -336,10 +192,6 @@ func (s *resolveSuite) expectCharmFallbackCall(curl *charm.URL, in, out csparams
 	).Return(curl, out, []string{"bionic", "focal"}, err)
```

Comment:
expectCharmFallbackCall and expectCharmFallbackResolutionCall can be removed. Then charmRepo is no longer needed in the resolverSuite. And the mockgen line can be updated to remove CharmrepoForDeploy, before re-generating the mock file.